pkg/util/atomic_copier.go

```diff
@@ -0,0 +1,132 @@
+package util
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"k8s.io/klog/v2"
+)
+
+var (
+	cpArgs = []string{
+		"--verbose",
+		"--recursive",
+		"--preserve",
+		"--reflink=auto",
+	}
+)
+
+// AtomicCopier performs a two operation: copies source path to an intermediate location,
```

Comment:
To be more precise, it's not any intermediate location. It's the same destination path, but with temporary name. Can we make it clearer?